### ISSUE TITLE
[HOLD] [$250] BUG: Fab menu shows when sidebar is opened reported by @mdneyazahmad

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#11708]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Go to https://staging.new.expensify.com/
2. Click on the fab button
3. Press command + k

## Expected Result:
Fab menu should be closed

## Actual Result:
Fab menu is not closed, and needs to perform any action on the sidebar to make it disappear

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Web
- Desktop App

**Version Number:** 1.2.12-2
**Reproducible in staging?:** y
**Reproducible in production?:** y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/194931422-d8164724-3d53-490b-bd3a-b00606a40ce6.mov
https://user-images.githubusercontent.com/43996225/194931437-5d49115e-cc84-4f34-bbb5-96d4270edf52.mp4

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @mdneyazahmad
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1665394830111229

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by kevinksullivan** [comment]
https://www.upwork.com/jobs/~018bcb6fe8dcaa0c58

**Comment by MarkCSmith** [comment]
Proposal:  Add a navigation event listener inside `src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.js` to detect when a different screen gains focus.  Call the existing `hideCreateMenu()` function inside the listener.

This will fix the Cmd+K (search) case as well as others such as Cmd+Shift+K (new group).

**Comment by Beamanator** [comment]
Hmm ya this issue is "similar" ish to https://github.com/Expensify/App/issues/11229, but not exactly the same - @flodnv think we should combine them or leave them separate?

**Comment by flodnv** [comment]
If we think that https://github.com/Expensify/App/issues/11229 solves this problem (which it might?), then we should put this on HOLD, probably?

**Comment by Beamanator** [comment]
Makes sense to me 👍

**Comment by flodnv** [comment]
Waiting on @parasharrajat to review the proposal here: https://github.com/Expensify/App/issues/11229#issuecomment-1286861407

**Comment by flodnv** [comment]
Still HOLDing https://github.com/Expensify/App/issues/11229#issuecomment-1306099255

**Comment by flodnv** [comment]
Still HOLDing, PR is up now.

**Comment by trjExpensify** [comment]
👋 the other PR attached to #11229 fixed this, so closing this issue out. 

@mdneyazahmad I've paid you for reporting this issue. 👍
